src/plugins/.common/structures/DataHandling/DatabaseHandler.js

```diff
@@ -1,13 +1,246 @@
 const quickDB = require("quick.db");
+const { Guild, Channel } = require("./../../../../../api/models/index.js");
 
 class DatabaseHandler {
   constructor() {
+    this.guild = new GuildHandler();
+    this.invite = new InviteHandler();
+    this.channels = new ChannelsHandler();
     this.bean = new BeanHandler();
     this.reminder = new ReminderHandler();
-    this.invite = new InviteHandler();
     this.thread = new ThreadHandler();
     this.currency = new CurrencyHandler();
-    this.newUserRoles = new NewUserRolesHandler();
+  }
+}
+
+class GuildHandler {
+  constructor() {}
+
+  async createGuild(guildId) {
+    console.log(`Creating guild with id: ${guildId}`);
+    if (this.getGuild(guildId, false) !== null) return false;
+    return await Guild.create(
+      { discordId: guildId },
+      {
+        include: Guild.Channel,
+      }
+    );
+  }
+
+  async deleteGuild(guildId) {
+    console.log(`Deleting guild with id: ${guildId}`);
```

Comment:
Is there a way to disable these logs in production?

Reply:
I just removed the log